js/showError.js

```diff
@@ -0,0 +1,16 @@
+import makeDOM from "./makeDOM.js";
+
+const showError = (tag, message) => {
```

Comment:
showError가 사용되는 곳을 보니, 이렇게 표현하면 좋겠습니다
```suggestion
const showError = (inputElement, message) => {
```